src/frontend/views/status-guidance.njk

```diff
@@ -0,0 +1,34 @@
+{% macro statusGuidance() %}
+  <p class='govuk-body'>
+    Common banner states:
+    <ul class='govuk-list govuk-list--bullet'>
+      <li>
+        Invoice within threshold (green) - the discrepancy between measured event data and the invoice is within the threshold tolerance of of -1% and 1%. This invoice can be paid.
+      </li>
+      <li>
+        Invoice above threshold (red) - the discrepancy between measured event data and the invoice is above the threshold of 1%. The vendor may have overcharged within the invoice. Contact the Data & Analytics team to confirm measured data and the vendor to confirm the invoice.
+      </li>
+      <li>
+        Invoice below threshold (blue) - the discrepancy between measured event data and the invoice is below is below the threshold of -1%. The vendor may have undercharged within the invoice. Contact the Data & Analytics team to confirm measured data and the vendor to confirm the invoice.
+      </li>
+    </ul>
+  </p>
```

Comment:
From https://www.quackit.com/html/tags/html_p_tag.cfm#:~:text=List%20Elements%20%26%20the,inside%20the%20tag.:

"List elements cannot be children of a <p> element. Therefore, you cannot place [ul] or [ol] tags inside the <p> tag."